src/components/radios/_macro.spec.js

```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
     },
 };
 
+const EXAMPLE_RADIO_ITEM_WITH_PRESET_TEXTAREA = {
```

Comment:
```suggestion
const EXAMPLE_RADIO_ITEM_TEXTAREA_WITH_PRESET_VALUE = {
```
More consistent with other example names